clang/lib/Lex/InitHeaderSearch.cpp

```diff
@@ -312,7 +312,8 @@ bool InitHeaderSearch::ShouldAddDefaultIncludePaths(
     break;
 
   case llvm::Triple::UnknownOS:
-    if (triple.isWasm())
+    if (triple.isWasm() || ((triple.getVendor() == llvm::Triple::Apple) &&
+                            triple.isOSBinFormatMachO()))
```

Comment:
Maybe a bit much for only one call site, but `triple.isAppleMacho()` seems much nicer to me here. (For reference, `isTargetMachineMac` also has only one caller AFAICT)

Reply:
Ok sure, we can use it in the driver too